p2p/host/relaysvc/relay_test.go

```diff
@@ -0,0 +1,29 @@
+package relaysvc
+
+import (
+	"testing"
+
+	"github.com/libp2p/go-libp2p/core/network"
+	bhost "github.com/libp2p/go-libp2p/p2p/host/blank"
+	swarmt "github.com/libp2p/go-libp2p/p2p/net/swarm/testing"
+	"github.com/stretchr/testify/require"
+)
+
+func TestReachabilityChangeEvent(t *testing.T) {
+	h := bhost.NewBlankHost(swarmt.GenSwarm(t))
+	rmgr := NewRelayManager(h)
+	rmgr.reachabilityChanged(network.ReachabilityPublic)
```

Comment:
I have a slight preference for not calling unexported functions in tests (where possible). In this case, you could get the event bus (via `h.EventBus()`), and then emit reachability events.

Reply:
that's neat!

Reply:
this is fixed now